.circleci/config.yml

```
@@ -0,0 +1,79 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/ruby:2.4.5
```

Same as in the other mordor-auditing PR. We use 2.6 pretty much everywhere.

```suggestion
      - image: circleci/ruby:2.6.5
```